src/transformers/models/vit_mae/modeling_vit_mae.py

```diff
@@ -223,15 +223,18 @@ def __init__(self, config):
         self.num_patches = self.patch_embeddings.num_patches
         # fixed sin-cos embedding
         self.position_embeddings = nn.Parameter(
-            torch.zeros(1, self.num_patches + 1, config.hidden_size), requires_grad=False
+            torch.zeros(1, self.num_patches + 1, config.hidden_size),
+            requires_grad=False,
```

Comment:
@ydshieh I am unable to do the rebasing as mentioned in https://github.com/huggingface/transformers/pull/16255#issuecomment-1072669273 as @ariG23498 is the repo owner. 

Here's what I did before committing the changes:

* I cloned https://github.com/huggingface/transformers in a separate directory.
* Initialized a new Python virtual environment. 
* `cd`'d to this cloned `transformers` and rand `pip install -e .[quality]`. 
* After that from the forked `transformers` directory (from the `tf-mae` branch) I ran `make style`. 

Let me know if these steps are good to go.

Reply:
I am also new to this usecase. It seems logical and the effects are there. Therefore I think it is OK.